src/Form/SearchInput.js

```diff
@@ -0,0 +1,55 @@
+import React, { useRef, useState } from 'react';
+import { css } from 'styled-components';
+import { createComponent } from '../utils';
+import { createEasyInput } from './EasyInput';
+import Input from './Input';
+
+const Search = createComponent({
```

Comment:
Wondering if we can get `prop-types` with defaults for this component? Makes the docz a little nicer with a Props Table.